### PR TITLE
Added an `disable_output` argument to the misp-add-tag-to-attribute

### PR DESCRIPTION
**Pull request by israelpoli**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/content/issues/19632

## Description
This PR is a copy of this [PR](https://github.com/demisto/content/pull/20252)

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MISP/ReleaseNotes/2_1_20.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#### Integrations
+##### MISP v3
+- Updated the Docker image to: *demisto/pymisp2:1.0.0.50170*.
+- Added an *disable_output* argument to the ***misp-add-tag-to-attribute***.
```
Comment (dorschw) [inline Packs/MISP/ReleaseNotes/2_1_20.md:4]:
```suggestion
- Added the *disable_output* argument to the ***misp-add-tag-to-attribute*** command. When manually set to `true`, the command runs faster.
```

File: Packs/MISP/Integrations/MISPV3/MISPV3.yml [hunk 2]
```diff
@@ -1674,6 +1674,15 @@ script:
       - 'false'
       required: false
       secret: false
+    - name: disable_output
+      description: if true, the attribute information will not be displayed in the
```
Comment (ShirleyDenkberg) [inline Packs/MISP/Integrations/MISPV3/MISPV3.yml:1678]:
```suggestion
      description: If true, the attribute information will not be displayed in the
```

File: Packs/MISP/Integrations/MISPV3/MISPV3.yml [hunk 3]
```diff
@@ -1674,6 +1674,15 @@ script:
...
+        response, reducing the runtime
```
Comment (ShirleyDenkberg) [inline Packs/MISP/Integrations/MISPV3/MISPV3.yml:1679]:
```suggestion
        response, reducing the runtime.
```

**Comment by ShirleyDenkberg** [comment]
@dorschw Doc review completed.

File: Packs/MISP/Integrations/MISPV3/MISPV3_test.py [hunk 4]
```diff
@@ -719,3 +719,51 @@ def test_add_msg_email_object(mocker):
...
+            {
+                'readable_output': 'Tag test_tag has been successfully added to event test_uuid',
+                'outputs': {'test': 'test'},
+                'outputs_prefix': 'MISP.Event',
+            }
+        )
+    ]
+)
+def test_add_tag(demisto_args: dict, is_attribute: bool, expected_result: dict, mocker):
+
```
Comment (dorschw) [inline Packs/MISP/Integrations/MISPV3/MISPV3_test.py:757]: please add a _given-when-then_ description to the test before merging

File: Packs/MISP/Integrations/MISPV3/MISPV3_test.py [hunk 5]
```diff
@@ -719,3 +719,51 @@ def test_add_msg_email_object(mocker):
...
+    mock_misp(mocker)
+    from MISPV3 import add_tag
+    from pymisp import ExpandedPyMISP
+    mocker.patch.object(ExpandedPyMISP, 'tag', return_value={})
+    mocker.patch.object(ExpandedPyMISP, 'search', return_value={})
+    mocker.patch('MISPV3.build_attributes_search_response', return_value={'test': 'test'})
+    mocker.patch('MISPV3.build_events_search_response', return_value={'test': 'test'})
+    result = add_tag(demisto_args, is_attribute=is_attribute)
+
```
Comment (dorschw) [inline Packs/MISP/Integrations/MISPV3/MISPV3_test.py:766]: please assert that `PYMISP.search` is called if and only if necessary. 
You can add a `mocked_search =` prefix to line where it's mocked, then `assert bool(mocked_search.call_count)` (or something like that) `== not disable_output`

File: Packs/MISP/Integrations/MISPV3/MISPV3_test.py [hunk 6]
```diff
@@ -719,3 +719,61 @@ def test_add_msg_email_object(mocker):
...
+    mock_misp(mocker)
+    from MISPV3 import add_tag
+    from pymisp import ExpandedPyMISP
+    mocker.patch.object(ExpandedPyMISP, 'tag', return_value={})
+    mocked_search = mocker.patch.object(ExpandedPyMISP, 'search', return_value={})
+    mocker.patch('MISPV3.build_attributes_search_response', return_value={'test': 'test'})
+    mocker.patch('MISPV3.build_events_search_response', return_value={'test': 'test'})
+    result = add_tag(demisto_args, is_attribute=is_attribute)
+
+    assert bool(mocked_search.call_count) == (is_attribute and not demisto_args['disable_output']) or not is_attribute
```
Comment (dorschw) [inline Packs/MISP/Integrations/MISPV3/MISPV3_test.py:776]:
```suggestion
    assert bool(mocked_search.call_count) == ((is_attribute and not demisto_args['disable_output']) or not is_attribute)
```

**Review by dorschw — APPROVED** [review]

**Pull request merged**